proxmox/resource_vm_qemu.go

```diff
@@ -1926,11 +1927,11 @@ func initConnInfo(
 	}
 	log.Print("[DEBUG][initConnInfo] trying to find IP address of first network card")
 
+	interfaces, err := client.GetVmAgentNetworkInterfaces(vmr)
```

Comment:
if this fails the first time then it will break any other check later, easy to happen thought, I have a lot of vm failed in terraform but ok on proxmox because of that, especially in batch of 2+ vms

Reply:
Alright, i moved it back to it old position and the old err != nil check handels it.

Reply:
interfaces will be ok if the agent will come up at same point, so the code should work unless we reach the timeout limit